src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -1550,6 +1566,9 @@ public static BlittableJsonReaderObject WriteDummyDocumentForAttachment(JsonOper
                 ["Hash"] = details.Hash,
                 ["ContentType"] = string.Empty,
                 ["Size"] = details.Size,
+                [nameof(AttachmentName.Flags)] = AttachmentFlags.None,
+                [nameof(AttachmentName.RetireAt)] = null,
+                [nameof(AttachmentName.Collection)] = "@files"
```

Comment:
looks like a const

Reply:
this is what we use 6 lines down, but I will make it a constant